test-case/manual/DPMST.md

```diff
@@ -0,0 +1,67 @@
+# DPMST test
+DisplayPort Multi-Stream Transport is a manual test.
+
+# Preconditions
+1. Hardware requirements met for each DPMST case
+
+# Test Description
+* Test output from one test unit to two separate monitors with audio output via
+  DP (as well as with USB-C and HDMI adapters if monitors have those inputs
+  available).
+
+# 1) Single DP -> Multi DP hub Test Case:
+## Hardware Requirements
+1. One DisplayPort hub = single DP -> multi-DP
```

Comment:
Could you change this? It should be some DP-MST hub. Normal hub will not have this feature. @libinyang could you help to clarify?

Reply:
@xiulipan The info I received from @ClarexZhou when I asked if there was a specific type of hub needed was "I think any DP hub (support one to x) should be ok." If this isn't correct, it will definitely need to be updated.

Reply:
Addressed in latest update.